test/variants/service/VariantService.spec.ts

```diff
@@ -425,5 +426,52 @@ describe('VariantsService', () => {
         });
       });
     });
+
+    it(`And valid businessId with multiple pages of variants
+        and the singlePage option set to true
+        When retrieving variants
+        Then return resolved promise with a single page of variants`, () => {
+      const mockedAxiosAdapter = new axiosMockAdapter(axios);
+      mockedAxiosAdapter.onGet(`${Environment.STAGING}/${Version.V1}/products/variants`).reply(200, {
+        variantSearchResults: generateVariantSearchDetails(250),
+        pagination: {
+          lastPage: false
```

Comment:
Can you mock out the rest of pagination response:
`lastKey?: string;
  lastPage: boolean;
  nextPageRef?: string;
  pageSize: number;
  totalItems?: number;`

Those fields are important to be able to pagination so we should just ensure coverage here